app/src/main/java/com/github/polypoly/app/base/game/location/LocationBet.kt

```diff
@@ -0,0 +1,20 @@
+package com.github.polypoly.app.base.game.location
+
+import com.github.polypoly.app.base.game.Player
+import kotlin.random.Random
+
+/**
+ * A class that represent a bet on a location
+ * @property player The player who made the bet
+ * @property amount The amount of money bet
+ * @property randomNumber The random number generated by the player to know if he/she has won the
+ * bet in the case of a tie
+ * @property timeOfTheBet The time of the bet in Unix time (i.e. the number of milliseconds since
+ * January 1, 1970, 00:00:00 GMT)
+ */
+data class LocationBet (
+    val player: Player = Player(),
+    val amount: Int = 100,
+    val randomNumber: Float = Random.nextFloat(),
+    val timeOfTheBet: Long = System.currentTimeMillis(),
```

Comment:
As we discussed together, is it really useful to remember the time of the bet?

Reply:
Yes because if the two random numbers are equal, we use the time of the bet to decide between the two players, so that we are sure that all the players have the same winner

Reply:
We discussed it already and we agreed that this way is a bit "overkill" as the probability to have the same bet AND the same random number is very small, let's always say that the bet number 1 (stored in the DB) wins. Having a time for each bet might be too much, what do you think?

Reply:
_After a discussion I agree that the time can be stored as we're not sure that the order is the DB is deterministic_